<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 255);</string>
  </property>
  <widget class="QLabel" name="movimentacao">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Movimentação</string>
   </property>
  </widget>
  <widget class="QLabel" name="descricao">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Descrição</string>
   </property>
  </widget>
  <widget class="QLabel" name="valor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Valor</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="caixaDataMovimentacao">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Finança Pessoal</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataDaProducao">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
  </widget>
  <widget class="QComboBox" name="caixaMovimentacao">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true">Selecione</string>
   </property>
   <item>
    <property name="text">
     <string>Selecione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Entrada</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Saída</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="categoria">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Categoria</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="caixaDescricao">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="caixaValor">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="caixaCategoria">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string notr="true">Selecione</string>
   </property>
   <item>
    <property name="text">
     <string>Selecione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Conta Corrente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Conta Popança</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cartão de Crédito</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cartão de Débito</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="botaoCadastrar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);
background-color: rgb(12, 117, 255);
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
